src/mkdocstrings/handlers/__init__.py

```diff
@@ -225,14 +230,16 @@ def get_handler(name: str, theme: str, custom_templates: Optional[str] = None) -
         name: The name of the handler. Really, it's the name of the Python module holding it.
         theme: The name of the theme to use.
         custom_templates: Directory containing custom templates.
+        get_handler_config: Callable that accepts a name str and returns a dict (which should be the config for the handler)
 
     Returns:
         An instance of a subclass of [`BaseHandler`][mkdocstrings.handlers.BaseHandler],
         as instantiated by the `get_handler` method of the handler's module.
     """
     if name not in HANDLERS_CACHE:
         module = importlib.import_module(f"mkdocstrings.handlers.{name}")
-        HANDLERS_CACHE[name] = module.get_handler(theme, custom_templates)
+        setup_cmd_list = get_handler_config(name).get("setup_commands", None) if get_handler_config else None
+        HANDLERS_CACHE[name] = module.get_handler(theme, custom_templates, setup_cmd_list)
```

Comment:
Just pass `handler_config` (see previous comments) as `**config` to `module.get_handler`.

I'd like to avoid hardcoding the options here, in case we have many more handlers later with each its own options.